examples/02-ui-components/formatting-toolbar-block-type-items/App.tsx

```diff
@@ -11,7 +11,34 @@ import { RiImage2Fill } from "react-icons/ri";
 
 export default function App() {
   // Creates a new editor instance.
-  const editor = useCreateBlockNote();
+  const editor = useCreateBlockNote({
```

Comment:
re this example, it's not really useful to be able to change something to an image right? Can we come up with something else?